bin/get_config.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
```

Comment:
I'm thinking that this belongs in https://github.com/perftool-incubator/toolbox/tree/main/bin, right?  Eventually we are going want to be able to extract things from this configuration file in a bunch of different places and putting a tool like this in toolbox is the best way to ensure it's available.

We should also maybe use a slighly more descriptive name, `get_config.py` is pretty vague.

Reply:
agreed